Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+        gemSummaryLineEdit->setPlaceholderText(tr("A short description of your Gem"));
+        secondScreen->addWidget(m_gemSummary);
+
+        m_requirements = new FormLineEditWidget(tr("Requirements"), "");
+        QLineEdit* requirementsLineEdit = m_requirements->lineEdit();
+        requirementsLineEdit->setPlaceholderText(tr("Notice of any requirements your Gem. i.e. This requires X other gem"));
+        secondScreen->addWidget(m_requirements);
+
+        m_license = new FormLineEditWidget(tr("License*"), "");
+        QLineEdit* licenseLineEdit = m_license->lineEdit();
+        licenseLineEdit->setPlaceholderText(tr("License uses goes here: i.e. Apache-2.0 or MIT"));
+        secondScreen->addWidget(m_license);
+
+        m_licenseURL = new FormLineEditWidget(tr("License URL"), "");
+        QLineEdit* licenseURLLineEdit = m_licenseURL->lineEdit();
+        licenseURLLineEdit->setPlaceholderText(tr("Link to the license web site i.e. https://opensource.org/licenses/Apache-2.0"));
+        secondScreen->addWidget(m_licenseURL);
+
+        m_origin = new FormLineEditWidget(tr("Origin"), "");
+        QLineEdit* originLineEdit = m_origin->lineEdit();
+        originLineEdit->setPlaceholderText(tr("The name of the originator goes. i.e. XYZ Inc."));
+        secondScreen->addWidget(m_origin);
+
+        m_originURL = new FormLineEditWidget(tr("Origin URL"), "");
+        QLineEdit* originURLLineEdit = m_originURL->lineEdit();
+        originURLLineEdit->setPlaceholderText(tr("The primary repo for your Gem. i.e. http://www.mydomain.com"));
+        secondScreen->addWidget(m_originURL);
+
+        m_firstDropdown = new FormComboBoxWidget(tr("Global Gem Tag*"));
+        AddDropdownActions(m_firstDropdown);
+        secondScreen->addWidget(m_firstDropdown);
+
+        m_secondDropdown = new FormComboBoxWidget(tr("Optional Global Gem Tag"));
+        AddDropdownActions(m_secondDropdown);
+        secondScreen->addWidget(m_secondDropdown);
+
+        m_thirdDropdown = new FormComboBoxWidget(tr("Optional Global Gem Tag"));
+        AddDropdownActions(m_thirdDropdown);
+        secondScreen->addWidget(m_thirdDropdown);
+
+        m_userDefinedGemTags = new FormLineEditWidget(tr("User-defined Gem Tags (Comma separated list)"), "");
+        secondScreen->addWidget(m_userDefinedGemTags);
+
+        m_gemLocation = new FormFolderBrowseEditWidget(tr("Gem Location"), "");
+        QLineEdit* gemLocationLineEdit = m_gemLocation->lineEdit();
+        const char* gemLocationPlaceholderText = R"(C:\O3DE\Project\Gems\YourGem)";
+        gemLocationLineEdit->setPlaceholderText(tr(gemLocationPlaceholderText));
+        secondScreen->addWidget(m_gemLocation);
+        
+        m_gemIconPath = new FormFolderBrowseEditWidget(tr("Gem Icon Path"), "");
+        QLineEdit* gemIconPathLineEdit = m_gemIconPath->lineEdit();
+        gemIconPathLineEdit->setPlaceholderText(tr("Select Gem icon path"));
+        secondScreen->addWidget(m_gemIconPath);
```

Comment:
This should default to "preview.png" and it should not use a Browse widget because it must be a relative path - or we need to modify Browse widget to only return a relative path.